src/main/java/edu/ucsb/cs156/spring/backenddemo/services/JokeQueryService.java

```diff
@@ -1,23 +1,48 @@
 package edu.ucsb.cs156.spring.backenddemo.services;
 
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
 import org.springframework.boot.web.client.RestTemplateBuilder;
 import org.springframework.stereotype.Service;
 import org.springframework.web.client.HttpClientErrorException;
 import org.springframework.web.client.RestTemplate;
 
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+
 
+@Slf4j
 @Service
 public class JokeQueryService {
+    ObjectMapper mapper = new ObjectMapper();
 
     private final RestTemplate restTemplate;
 
     public JokeQueryService(RestTemplateBuilder restTemplateBuilder) {
         restTemplate = restTemplateBuilder.build();
     }
 
-    public static final String ENDPOINT = "";
+    public static final String ENDPOINT = "https://v2.jokeapi.dev/joke/{category}?amount={numJokes}";
 
     public String getJSON(String category, int numJokes) throws HttpClientErrorException {
```

Comment:
Not the change, but just noting: is `numJokes` int or string? Why was it given as an `int` but the example Swagger UI website lists it as a `string`

http://team01.dokku-00.cs.ucsb.edu/swagger-ui/index.html#/joke-controller/getJokesUsingGET